### PR TITLE
Do not repeat the name of power unit.

### PR DESCRIPTION
**Pull request by MaxJPRey**
Fix issue #349 .
The string returned when editing the source was repeating the power unit.

### CODE REVIEW

**Review by maxcapodi78 — APPROVED** [review]

File: pyaedt/hfss.py [hunk 1]
```diff
@@ -2373,12 +2373,12 @@ def edit_source(self, portandmode, powerin, phase="0deg"):
...
         ...                                              2, "SheetWavePort", True)
         >>> hfss.edit_source("SheetWavePort" + ":1", "10W")
-        pyaedt Info: Setting up power to Eigenmode 10WWatt
+        pyaedt Info: Setting up power to Eigenmode 10W
         True
 
         """
 
-        self._messenger.add_info_message("Setting up power to Eigenmode " + powerin + "Watt")
+        self._messenger.add_info_message("Setting up power to Eigenmode " + powerin)
```
Comment (PipKat) [inline pyaedt/hfss.py:2381]:
```suggestion
        self._messenger.add_info_message("Setting up power to Eigenmode " + power in.)
```

Comment (PipKat) [inline pyaedt/hfss.py:2381]:
```suggestion
        self._messenger.add_info_message("Setting up power to Eigenmode " + ``powerin``.)
```

Comment (PipKat) [reply pyaedt/hfss.py:2381]: @MaxJPRey I'm not sure if the literal tagging is acceptable within a message, but if so, it seems more readable, especially when concluding the sentence with a period.

**Review by PipKat — APPROVED** [review]

File: pyaedt/hfss.py (hunk 1 above)
Comment (maxcapodi78) [reply pyaedt/hfss.py:2381]: as part of a string passed to an argument we should not use ``. Original command from @MaxJPRey was correct

**Pull request merged**